torch/_dynamo/variables/misc.py

```diff
@@ -744,7 +802,17 @@ def call_method(
         from ..trace_rules import is_callable_allowed
         from .builder import wrap_fx_proxy
 
-        if name == "apply":
+        # There are two cases to handle the apply method of an autograd function:
+        # 1. If the autograd function is not vmapified:
+        #    - We can directly handle it by either treating it as allow_in_graph or
+        #      wrapping it as an AutogradFunctionApplyVariable HOP.
```

Comment:
I don't think we ever allow_in_graph an autograd.Function unless the user has explicitly used allow_in_graph?

Reply:
Yes, here it refers users explicitly used allow_in_graph. I'll update the comment to clarify this.